scripts/puritan.lic

```diff
@@ -112,13 +112,13 @@ end
 ##
 module Puritan
   module Outcomes
-    ALL       = /what were you|cannot|must be holding|shatter|song misfires/i
     MAX       = /cannot be/
     SHATTERED = /shatter/
     MISFIRE   = /misfire/
     STRAIN    = /you hear it crack loudly and strain/
     PURE      = /begins to twist and turn as the very essence flows into it filling the imperfections/
     IMPROVED  = /more perfect|what were you|improves somewhat|crack|shatter|cannot be|must be holding|smoother and more pure in color/
+    ALL       = Regexp.union(MAX, SHATTERED, MISFIRE, STRAIN, PURE, IMPROVED)
```

Comment:
I couldn't find anywhere the existing constant was used. I did this with reflection at first (`Puritan::Outcome.constants...`) but it felt way too cute.